static/bidder-params/revcontent.json

```diff
@@ -0,0 +1,9 @@
+
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Revcontent Adapter Params",
+  "description": "A schema which validates params accepted by the Revcontent adapter",
+  "type": "object",
+  "properties": {},
```

Comment:
Does your adapter truly have no bidder parameters? If so, not a problem.

If you do have bidder parameters but aren't defining them here because you pass the OpenRTB request exactly as is, please consider adding them here to allow PBS to enforce the parameters are provided and prevent outgoing traffic to your bidding server if the request is invalid.

Reply:
Do you mean things like `bidfloor`? Then yes, I suppose I'll have to look at this. Thanks!

Reply:
We really just need to look at:
(request.App.name OR request.Site.domain) AND request.Device, and everything else is optional on our end.

Do we need to enforce these in a check following this example: https://github.com/prebid/prebid-server/blob/master/adapters/pubnative/pubnative.go#L80-L88

Or is there a way to use the params.json to check those? We don't currently need any ext params or anything.

Reply:
I wouldn't say you need to, but I do highly recommend it. The earlier we can disqualify a bid request the less resources we all have to use to accomodate it. In this case, PBS hosts wouldn't need to reach out to your bidding servers for requests which are known to be invalid.

This must be coded in the adapter, as you linked to for pubnative. The params.json file is just to check bidder extensions not define within the OpenRTB spec.

Thank you for the details. This json file is fine as is. I do encourage you to add the checks you mentioned along with new json tests cases to cover the new logic.